esp/src/eclwatch/LFDetailsWidget.js

```diff
@@ -116,6 +116,7 @@ define([
             var context = this;
             var origOnOpen = this.desprayTooltiopDialog.onOpen;
             this.desprayTooltiopDialog.onOpen = function () {
+                var targetRow;
```

Comment:
Is this part of the webpack improvements?

Reply:
Its fixing the linting errors